docs/smes.md

```diff
@@ -0,0 +1,65 @@
+# Subject Matter Experts (SMEs)
+
+A given subject matter expert is not necessarily the most knowledgeable in the area, but they are the one who probably knows the most about it when it was originally done or has a responsibility to expand their knowledge to know about the area going forward. Contacting them first will help with delegation of responsibilities at the Dashboard level.
+
+This will detail out former (or current) feature leads, area leads (has a responsibility to understand the area), as well as any other notable position in relation to the area. If you need to talk to someone or ping someone for a review, this information should help you determine who.
+
+Below there will be some terms like “previous” and “backup”, these are for additional context. The way you can read each are as follows:
+- **previous** – the initial SME in the area. If you need legacy context, this person may be able to help
+- **backup** – a good person to lean on if there is a need for any 2nd opinions, for bouncing ideas off of, or any larger discussion about direction
+- **and** – Ping both during conversations – could be onboarding, could be a need to share information, best get both people involved at the same time
+
+General Dashboard ownership
+- Infrastructure / direction
+    - Architect: `Andrew` ([andrewballantyne])
+    - General UX: `Kyle` ([kywalker-rh])
+    - App Text: `Katie` ([kaedward])
+- Testing (Integration, Unit, etc)
+    - Area lead: `Gage` ([Gkrumbach07])
+- Performance
+    - Area lead: `Lucas` ([lucferbux])
+
+Dashboard features
+- Data Science Projects
+    - Feature lead: `Andrew` ([andrewballantyne])
+    - UX: `Kyle` ([kywalker-rh]) **and** `Kun` ([xianli123])
+- Data Science Pipelines
+    - Feature lead: `Andrew` ([andrewballantyne])
+    - UX: `Yan` ([yannnz])
+        - Previous: `Kyle` ([kywalker-rh])
+- Explainability, Bias
+    - Feature lead: `Alex` ([alexcreasy])
+    - UX: `Vince` ([vconzola])
+- Model Serving (Custom runtimes, general Model Serving)
+    - Feature lead: `Lucas` ([lucferbux])
+    - UX: `Vince` ([vconzola])
+- Model Serving - Performance metrics
+    - Feature lead: `Andrew` ([andrewballantyne])
+    - UX: `Vince` ([vconzola])
+- BYON - Custom Notebook Images
+    - Feature lead: `Juntao` ([DaoDaoNoCode])
+        - Backup: `Andrew` ([andrewballantyne])
+    - UX: `Vince` ([vconzola])
+- Habana / Accelerators
+    - Feature lead: `Gage` ([Gkrumbach07]);
+        - Backup: `Andrew` ([andrewballantyne])
+    - UX: `Yan` ([yannnz])
+- Model Registry
+    - Feature lead: TBD
+    - UX: `Sim` ([simrandhaliw])  **and** `Haley` ([yih-wang])
+
+<!-- Developers -->
```

Comment:
Does this need updating with the full list of dashboard devs?

Reply:
It's hidden text, just so the url doesn't need to be repeated over and over. Consider these variables in play, I just separated them so it would be easier to see if one was already there before you added a new one. If they are not part of SMEs, they don't really need to be here yet (imo)